src/Languages/Editor/Impl/Microsoft.Languages.Editor.csproj

```diff
@@ -156,6 +156,10 @@
       <Project>{8d408909-459f-4853-a36c-745118f99869}</Project>
       <Name>Microsoft.Common.Core</Name>
     </ProjectReference>
+    <ProjectReference Include="..\..\..\Windows\Core\Impl\Microsoft.Windows.Core.csproj">
```

Comment:
I changed editor stuff to

`Microsoft.R.Editor` - common code
`Microsoft.R.Editor.Windows` - OS specific

namespace is the same in both and is `Microsoft.R.Editor`

So - `Microsoft.R.Common.Core` & `Microsoft.R.Common.Core.Windows` ?